source/lib/thunderbird.js

```diff
@@ -14,26 +20,10 @@ function showSimpleNewMessageNotification(isRSS) {
     var title = isRSS ? _("New_article") : _("New_message");
     var text = _("Number_of_unread_messages") + " " + count;
 
```

Comment:
I put all of this in a separate function called `showNotification`, since the exact same code is called by more than one function.